api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -9277,7 +9277,7 @@
   },
   "openapi": "3.0.0",
   "paths": {
-    "/api/v1/": {
+    "/api/v1": {
```

Comment:
this change to all paths in all openapi docs is questionable, is there a way to avoid a user-facing change in our served openapi here?

Reply:
I think that's related to https://github.com/emicklei/go-restful/issues/519, see https://github.com/kubernetes/kubernetes/pull/115067